art-blocks-engine-onboarding/art-blocks-engine-101/Mobile-Minter.md

```diff
@@ -0,0 +1,139 @@
+
+Web-enabled TVs or displays can showcase a real-time view of the latest tokens minted through the Mobile Minter. The Art Blocks documentation site offers an [overview of configuration options for the live viewer](https://docs.artblocks.io/creator-docs/art-blocks-api/artblocks-viewer/#artblocks-viewer).
+
+### Example Embed
+
+You can embed [live.artblocks.io](http://live.artblocks.io) using an iframe on a web page containing your organization’s branding.
+
+[Example embed of [live.artblocks.io](http://live.artblocks.io) without any configuration parameters ([docs](https://docs.artblocks.io/creator-docs/art-blocks-api/artblocks-viewer/#artblocks-viewer))](https://live.artblocks.io/)
+
+Example embed of [live.artblocks.io](http://live.artblocks.io) without any configuration parameters ([docs](https://docs.artblocks.io/creator-docs/art-blocks-api/artblocks-viewer/#artblocks-viewer))
+
+---
+
+## Frequently Asked Questions
+
+### Can the Mobile Minter be used with projects that are not paused?
+
+Yes, the Mobile Minter can be used for active projects. However, pausing the project and transferring artist ownership to the app restricts minting exclusively through the app. If a project remains unpaused, anyone online can mint tokens.
+
+### Can users pay for gas fees themselves?
+
+Currently, gas fees must be pre-funded by the artist or the organization using the Mobile Minter app, through the Mobile Minter's hot wallet.
+
+### Can my prepaid gas fee balance be returned to me?
+
+Yes, please contact us to arrange the return of any remaining funds to the original depositing wallet.
+
+### Can multiple projects be managed simultaneously through the Mobile Minter app?
+
+Yes, the Mobile Minter app allows you to manage multiple projects for in-person events. You can easily switch between projects during the event, ensuring a seamless minting experience for attendees.
+
+### What currencies are supported for payments in the Mobile Minter app?
+
+The Mobile Minter app processes fiat payments via Stripe Terminal, which supports a variety of currencies. The available currencies depend on your Stripe account and the country where your business operates. You will be the merchant of record, and Art Blocks will collect its platform fees in ETH or USD. For a list of supported currencies for your customers, please refer to the [Stripe documentation](https://stripe.com/docs/currencies).
+
+### Is it possible to customize the appearance of the Mobile Minter app for my event?
+
+While the Mobile Minter app does not offer customization, you can embed [live.artblocks.io](https://live.artblocks.io/) inside of a page that showcases your event's theme and branding. Please refer to the [Art Blocks documentation site](https://docs.artblocks.io/creator-docs/art-blocks-api/artblocks-viewer/#artblocks-viewer) for an overview of configuration options for the live viewer.
+
+### I have unspent ETH in the Mobile Minter's hot wallet. How can I retrieve the funds?
```

Comment:
this entry can be combined with "Can my prepaid gas fee balance be returned to me?"